index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Hello Micronaut!</title>
+    </head>
+    <body>
+        <!-- <h1 style="background-color: #000000; color: #7e02d1; text-align: center; font-size: 40px;">Hello Microverse!</h1> -->
+        <h1>Hello Microverse!</h1>
+    </body>
+</html>
```

Comment:
**[ OPTIONAL ]**

- [ ] I noticed you are using 4 spaces instead of 2 for indentation. It's best practice to use 2. Feel free to fix this to improve your code.
- [ ] Semantic HTML is another way to make our code more professional. You may add some semantic HTML here to fix this!